docs/tutorials/essentials/part-2-app-structure.md

```diff
@@ -280,7 +280,7 @@ console.log(newState)
 
 We said earlier that reducers must **always** follow some special rules:
 
-- They should only calculate the new state value based on the `state` and `action` arguments
+- They should only calculate the new state value based on the `state` and `action` arguments.
 - They are not allowed to modify the existing `state`. Instead, they must make _immutable updates_, by copying the existing `state` and making changes to the copied values.
 - They must not do any asynchronous logic or other "side effects"
```

Comment:
```suggestion
- They must not do any asynchronous logic or other "side effects".
```

Reply:
actually, it seems quite a few of the other list items also don't end in a full stop 😕